recipe/deploy/writable.php

```diff
@@ -22,7 +22,7 @@
         return;
     }
 
-    if ($httpUser === false && $mode !== 'chmod') {
+    if ($httpUser === false && ! in_array($mode, ['chgrp', 'chmod'], true)) {
```

Comment:
But you need http_user: `run("$sudo chgrp -RH $httpGroup $dirs", $runOpts);`

Reply:
No you don't: the full code of `chgrp` is:

https://github.com/deployphp/deployer/blob/62002e7a07fbcd4009d5695c85d5d997e6efd208/recipe/deploy/writable.php#L48-L56

And here the only variable used is `http_group`, no mention to `http_user`